### PR TITLE
Add Tap BPM

### PR DESCRIPTION
**Pull request by yurrriq**
"Find the beat of a song by tapping any key on your keyboard."

### CODE REVIEW

File: Casks/tap-bpm.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'tap-bpm' do
+  version :latest
+  sha256 :no_check # 'ed007722fea1b88561f3e3714e5d626f0a9ab9e9e3dc8027cc54a162b0985083'
```
Comment (vitorgalvao) [inline Casks/tap-bpm.rb]: Comment is useless, please remove it.

File: Casks/tap-bpm.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  url "http://www.drumbot.com/resources/tap_bpm_mac.zip", :referer => 'http://www.drumbot.com/projects/tap_bpm/'
```
Comment (vitorgalvao) [inline Casks/tap-bpm.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

Are you sure `:referer` is needed?

File: Casks/tap-bpm.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name 'Tap BPM'
+  homepage 'http://www.drumbot.com'
```
Comment (vitorgalvao) [inline Casks/tap-bpm.rb]: http://www.drumbot.com/projects/tap_bpm/ is more accurate.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 508ed77831f6d4c69aee222221fc62cf860eca28 to keep commit history cleaner. Your contribution is still credited to you.

**Comment by yurrriq** [comment]
:+1:

**Pull request closed without merging**